content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -173,8 +179,8 @@ Example using `/etc/hosts`:
 
 ## Running services on a mesh expansion machine
 
-* Configure a ServiceEntry. The ServiceEntry will contain the IP addresses, ports and labels of all
-VMs exposing a service.
+VMs are added to the mesh by configuring a ServiceEntry. The ServiceEntry will contain the IP 
```

Comment:
Passive voice, verb tense congruence and markup issues:
Add VMs to the mesh by configuring a \`ServiceEntry\`. The \`ServiceEntry\` contains the IP